### DISCUSSION TITLE
system crashes when using midi cc on cutoff..

### DISCUSSION THREAD

**Discussion by DvV74** [probonopd/MiniDexed#738] [General]
When i use midi cc controller on the cutoff, the whole synth crashes.  can someone verify this?

**Comment by Banana71** [comment]
Hi DvV74,

No problems with CC 74 and CC 71
I'm running the following version: https://github.com/probonopd/MiniDexed/pull/690#issuecomment-2241782517

https://github.com/user-attachments/assets/c72b924d-3bb6-44b2-b100-3de2129dad1c
